### PR TITLE
Add basic styles to about:about and created page w/ version information (about:brave)

### PR DESCRIPTION
**Pull request by bsclifton**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Add basic styles and version information to about:about

Fixes https://github.com/brave/browser-laptop/issues/463

Auditors: @bbondy, @bradleyrichter

## Test Plan
1. Launch Brave and visit about:about
2. Pull up versions from the "About Brave" message box: Help > About Brave (or on Mac, Brave > About Brave)
3. Compare the versions shown on the about:about page to the ones in the message box

## Screenshots
**The new look**
screen shot 2016-11-06 at 2 20 12 am

**Text is really easy to select for copy/paste**
screen shot 2016-11-06 at 2 22 27 am

**compare this to the current look**
screen shot 2016-11-06 at 2 35 14 am

### CODE REVIEW

File: js/about/about.js [hunk 1]
```diff
@@ -3,23 +3,63 @@
...
+require('../../node_modules/font-awesome/css/font-awesome.css')
+
+class AboutAbout extends React.Component {
+  constructor () {
+    super()
+    this.state = { versionInformation: Immutable.fromJS([]) }
+    ipc.on(messages.VERSION_INFORMATION_UPDATED, (e, versionInformation) => {
+      console.log('got message')
+      if (this.state.versionInformation.size === 0) {
+        console.log('updating')
```
Comment (bbondy) [inline js/about/about.js]: nit: logging

**Comment by bbondy** [comment]
This is great, I don't mind about:about being combined either. Can you get rid of the dialog though from the about menu and make it go here instead? Test main menu locations and hamburger menu location.

**Comment by bsclifton** [comment]
Maybe once there's more about pages, we can break this out into it's own about:version page; my initial thought was that (with limited info) it makes sense to show as many details about Brave as possible in one place :smile: We could possibly add more info we find valuable for troubleshooting, like OS version, Video drivers, etc.

Working on the fixes now...

**Comment by luixxiul** [comment]
Wouldn't it nice if we display the additional version info like "RC1" or "preview1"?

**Comment by bsclifton** [comment]
Good call, @luixxiul!  I captured that with https://github.com/brave/browser-laptop/issues/5462

**Comment by bsclifton** [comment]
Updated! @darkdh, if you have a moment, please check out the changes. There are two message boxes that were in `app/aboutDialog.js` which I moved into `app/importer.js`. They were being exported, but I changed them to local `const` because they're not used elsewhere.  I tested an import to verify it loads (but didn't try the one w/ a warning).

@bbondy picking `About Brave` now brings you to the about page :smile: The previous message box has been removed

**Comment by bsclifton** [comment]
Here we go- even more useful info for debugging :smile: 

screen shot 2016-11-07 at 11 34 30 am

**Comment by darkdh** [comment]
++ for the importer dialog, thx.

**Comment by bbondy** [comment]
cc @bradleyrichter do we want to combine the about page with about:about? Note other browsers keep them separate and we might double or triple our about:about pages in the next year. I'm fine either way.

File: js/about/entry.js [hunk 2]
```diff
@@ -6,50 +6,54 @@ const ipc = window.chrome.ipc
...
+    break
+  case 'about:preferences':
+    element = require('./preferences')
+    break
+  case 'about:safebrowsing':
+    element = require('./safebrowsing')
+    break
   case 'about:styles':
     element = require('./styles')
     break
```
Comment (bsclifton) [inline js/about/entry.js]: this is sorted alphabetical now

File: js/lib/appUrlUtil.js [hunk 3]
```diff
@@ -52,21 +52,22 @@ module.exports.getManifestUrl = function () {
...
+  'about:passwords': module.exports.getAppUrl('about-passwords.html'),
   'about:preferences': module.exports.getAppUrl('about-preferences.html'),
-  'about:config': module.exports.getAppUrl('about-config.html'),
-  'about:certerror': module.exports.getAppUrl('about-certerror.html'),
   'about:safebrowsing': module.exports.getAppUrl('about-safebrowsing.html'),
-  'about:passwords': module.exports.getAppUrl('about-passwords.html'),
-  'about:flash': module.exports.getAppUrl('about-flash.html'),
-  'about:error': module.exports.getAppUrl('about-error.html'),
-  'about:autofill': module.exports.getAppUrl('about-autofill.html'),
   'about:styles': module.exports.getAppUrl('about-styles.html')
```
Comment (bsclifton) [inline js/lib/appUrlUtil.js]: same here; alphabetical FTW :stuck_out_tongue:

**Comment by bbondy** [comment]
fancy with the sortable table 👍

**Comment by bbondy** [comment]
fannnntastic, thanks.

**Comment by srirambv** [comment]
@bsclifton doesn't brave/muon/channel/os needs to be capitalized??

**Comment by bsclifton** [comment]
@srirambv that is a good question! I didn't consider that- I'll do a follow up which fixes that :smile:

**Pull request merged**